### PR TITLE
Fix UPS tracking token issue

### PR DESCRIPTION
**Pull request by norkator**
Draft pr

### CODE REVIEW

**Review by ristomatti — APPROVED** [review]
Nice and neat!

File: app/src/main/java/com/nitramite/courier/UPSStrategy.java [hunk 1]
```diff
@@ -45,21 +51,35 @@ public ParcelObject execute(String parcelCode) {
...
+
             OkHttpClient client = new OkHttpClient();
 
-            final String json = "{\"Locale\":\"fi_FI\",\"TrackingNumber\":[\"" + parcelCode + "\"]}"; // TODO: make proper way
-            Log.i(TAG, json);
-            RequestBody body = RequestBody.create(json, JSON);
+            JSONArray trackingNumbers = new JSONArray();
+            trackingNumbers.put(parcelCode);
+            JSONObject requestBody = new JSONObject();
+            requestBody.put("Locale", "fi_FI");
```
Comment (ristomatti) [inline app/src/main/java/com/nitramite/courier/UPSStrategy.java:64]: Double checking - I assume this is matched with `loc=fi_FI` in `url ` and not `loc=en_US` in `tokenUrl`?

**Review by ristomatti — APPROVED** [review]

**Comment by ristomatti** [comment]
When will the update be released? It still could make it before my laptop arrives :smile:.

Not happy about the delay though. First it took >6 weeks longer for it to even ship and now the package has been stuck somewhere in China "due to reasons unrelated to UPS" according to the customer service. On Saturday it was still estimated to arrive yesterday. :unamused:

File: app/src/main/java/com/nitramite/courier/UPSStrategy.java (hunk 1 above)
Comment (norkator) [reply app/src/main/java/com/nitramite/courier/UPSStrategy.java:64]: Aaah yeah that is something I didn't check for but in this case getting token with en_US didn't change outcome

**Comment by norkator** [comment]
> When will the update be released? It still could make it before my laptop arrives 😄.
> 
> Not happy about the delay though. First it took >6 weeks longer for it to even ship and now the package has been stuck somewhere in China "due to reasons unrelated to UPS" according to the customer service. On Saturday it was still estimated to arrive yesterday. 😒

Since hurry... releasing now asap.

**Pull request merged**